google-cloud-clients/pom.xml

```diff
@@ -660,7 +660,7 @@
           <dependency>
             <groupId>com.puppycrawl.tools</groupId>
             <artifactId>checkstyle</artifactId>
-            <version>6.15</version>
+            <version>6.16</version>
```

Comment:
This doesn't really belong here, but needed to make this build on my machine at home. See #3577